src/main/java/teammates/common/datatransfer/FeedbackSessionResultsBundle.java

```diff
@@ -28,7 +28,7 @@
  * <br> * The basic {@link FeedbackSessionAttributes}
  * <br> * {@link List} of viewable responses as {@link FeedbackResponseAttributes} objects.
  */
-public class FeedbackSessionResultsBundle implements SessionResultsBundle {
```

Comment:
Unused interface (just take a look, it's an empty interface), and I'm not going to maintain the docs for something unused.